provider/pkg/gen/dotnet-templates/helm/v3/Chart.cs

```diff
@@ -25,410 +26,432 @@
-            public bool? IncludeTestHookResources { get; set; }
-            [JsonPropertyName("skip_crd_rendering")]
-            public bool? SkipCRDRendering { get; set; }
-            [JsonPropertyName("namespace")]
-            public string? Namespace { get; set; }
-            [JsonPropertyName("values")]
-            public ImmutableDictionary<string, object> Values { get; set; } = null!;
-            [JsonPropertyName("repo")]
-            public string? Repo { get; set; }
-            [JsonPropertyName("chart")]
-            public string? Chart { get; set; }
-            [JsonPropertyName("version")]
-            public string? Version { get; set; }
-            [JsonPropertyName("fetch_opts")]
-            public JsonOptsFetch? FetchOptions { get; set; }
-            [JsonPropertyName("path")]
-            public string? Path { get; set; }
-            [JsonPropertyName("release_name")]
-            public string? ReleaseName { get; set; }
-        }
-        internal class JsonOptsFetch
-        {
-            [JsonPropertyName("version")]
-            public string? Version { get; set; }
-            [JsonPropertyName("ca_file")]
-            public string? CAFile { get; set; }
-            [JsonPropertyName("cert_file")]
-            public string? CertFile { get; set; }
-            [JsonPropertyName("key_file")]
-            public string? KeyFile { get; set; }
-            [JsonPropertyName("destination")]
-            public string? Destination { get; set; }
-            [JsonPropertyName("keyring")]
-            public string? Keyring { get; set; }
-            [JsonPropertyName("password")]
-            public string? Password { get; set; }
-            [JsonPropertyName("repo")]
-            public string? Repo { get; set; }
-            [JsonPropertyName("untar_dir")]
-            public string? UntarDir { get; set; }
-            [JsonPropertyName("username")]
-            public string? Username { get; set; }
-            [JsonPropertyName("home")]
-            public string? Home { get; set; }
-            [JsonPropertyName("devel")]
-            public bool? Devel { get; set; }
-            [JsonPropertyName("prov")]
-            public bool? Prov { get; set; }
-            [JsonPropertyName("untar")]
-            public bool? Untar { get; set; }
-            [JsonPropertyName("verify")]
-            public bool? Verify { get; set; }
-        }
-    }
+		// Set skipAwait annotation on all chart resources.
+		internal ImmutableDictionary<string, object> SkipAwait(ImmutableDictionary<string, object> obj, CustomResourceOptions opts)
+		{
+			var metadata = (ImmutableDictionary<string, object>)obj["metadata"];
+			if (metadata.ContainsKey("annotations"))
+			{
+				metadata = metadata.SetItem("annotations", ((ImmutableDictionary<string, object>)metadata["annotations"]).SetItem("pulumi.com/skipAwait", "true"));
+                obj = obj.SetItem("metadata", metadata);
```

Comment:
Nit: something wrong with indentation on this line. Overall, it looks like you converted this whole file to tabs which isn't common for .NET code. I'd suggest using 4 spaces. (sorry, I hate to comment on tabs vs. spaces)

Reply:
Ah, something must have gotten messed up when I copy-pasted to GoLand.